nsis/src/nsis/file/NsisExecutable.java

```diff
@@ -1,11 +1,17 @@
 package nsis.file;
 
+import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import java.util.Arrays;
 
+import org.tukaani.xz.LZMAInputStream;
```

Comment:
Do you have some form of documentation listing all dependencies of your library. Where does one get `tukaani` from? (same for com.google.common.primitives.Bytes)

Reply:
I will try to add the dependence in the repo directly

Reply:
Readme updated and jar file added to the repo